ssl_cert_tracker/tasks.py

```diff
@@ -13,54 +13,210 @@
 :contact:    ali.rahmat@phsa.ca
 
 """
-
-import logging
-
 import xml.dom.minidom
-from django.core.exceptions import ObjectDoesNotExist
+
+from smtplib import SMTPConnectError
 from celery import shared_task
+from celery.utils.log import get_task_logger
+from celery.exceptions import MaxRetriesExceededError
 from libnmap.process import NmapProcess
 from orion_integration.lib import OrionSslNode
 
 from .db_helper import insert_into_certs_data
-
+from .lib import Email, expires_in, has_expired, is_not_yet_valid
 from .utils import process_xml_cert
-
-logging.basicConfig(filename='p_soc_auto.log', level=logging.DEBUG)
-
-
-@shared_task(rate_limit='0.5/s', queue='nmap')
+from .models import Subscription
+
+logger = get_task_logger(__name__)
+
+SSL_PROBE_OPTIONS = r'-Pn -p 443 --script ssl-cert'
```

Comment:
why did you use raw string literals here since we are not dealing with exacpe \ characters

Reply:
force of habit. when passing things to external programs (even if this is not really external but a wrapper around an external) i tend to use raw strings